### PR TITLE
Adds character limit of 72 to all the things

### PR DESCRIPTION
**Pull request by michaelck**

### CODE REVIEW

**Comment by michaelck** [comment]
This is on hold until a few legacy documents have been updated to meet the character limit.

**Comment by michaelck** [comment]
Request made to expand the character limit to additional content types.

File: wp/wp-admin/admin-header.php [hunk 1]
```diff
@@ -68,11 +68,15 @@
 wp_enqueue_script( 'svg-painter' );
 
 $admin_body_class = preg_replace('/[^a-z0-9_-]+/i', '-', $hook_suffix);
+
+$philaAllPostTypes = json_encode( array_values( get_post_types( '','names' ) ) );
```
Comment (karissademi) [inline wp/wp-admin/admin-header.php]: Let's move this to a non-core file. :)

**Comment by karissademi** [comment]
🚢 

Merge will commence once communication has been sent out of these changes.

**Pull request merged**